doc/usage/installation.rst

```diff
@@ -187,3 +187,45 @@ You can also download a snapshot of the Git repo in either `tar.gz`__ or
 __ https://github.com/sphinx-doc/sphinx
 __ https://github.com/sphinx-doc/sphinx/archive/master.tar.gz
 __ https://github.com/sphinx-doc/sphinx/archive/master.zip
+
+
+docker-sphinx
+-------------
+
+`docker-sphinx <https://hub.docker.com/r/sphinxdoc/sphinx>`_ is Docker images for Sphinx.
```

Comment:
How about this instead?

Docker images for Sphinx are published on the [Docker Hub](https://hub.docker.com/). There are two kind of images:

* [sphinxdoc/sphinx](https://hub.docker.com/repository/docker/sphinxdoc/sphinx)
* [sphinxdoc/sphinx-latexpdf](https://hub.docker.com/repository/docker/sphinxdoc/sphinx-latexpdf)

Former one is used for standard usage of Sphinx, and latter one is mainly used for PDF builds using LaTeX. Please choose one for your purpose.

.. note::

   sphinxdoc/sphinx-latexpdf contains TeXLive packages. So it is very large (over 2GB!).

Reply:
LGTM